server/settings_test.go

```diff
@@ -41,6 +42,57 @@ func TestSettings(t *testing.T) {
 
 			defer restoreSettingsDefaults(t)
 
+			t.Run("InvalidBothEnableAndDisableAlerting", func(t *testing.T) {
```

Comment:
Missing defaults check on lines 28-39

Reply:
And missing case with valid IA configuration update

Reply:
added